Assets/AppCenter/Properties/CustomDropDownPropertyAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using UnityEngine;
+
+[AttributeUsage(AttributeTargets.Field, AllowMultiple = true)]
```

Comment:
AllowMultiple?

Reply:
Yes, there are 2 attribute applied to property